README.md

```diff
@@ -12,8 +12,8 @@ This library contains reusable code for:
 ## Requirements
 
 - Boost 1.54
-- Ceres 1.12
-- Eigen 3.2.0
+- Ceres 1.13
+- Eigen 3.3
```

Comment:
Yes you're right. I want to support any 3.3, e.g. 3.3.0, but to claim that we'd have to use it on travis.